R/produce_source_acute_tests.R

```diff
@@ -1,4 +1,4 @@
-#' Produce the Acute tests
+#' Produce the Source Extract tests
 #'
 #' @param data new or old data for testing summary flags
 #' (data is from \code{\link{get_source_extract_path}})
```

Comment:
Add a description to the function documentation.

```suggestion
#' Produce the Source Extract tests
#'
#' @description Produce a set of tests which can be used by most
#' of the extracts. 
#' This will produce counts of various demographics
#' using [create_demog_test_flags()] counts of episodes for every `hbtreatcode`
#' using [create_hb_test_flags()], a total cost for each `hbtreatcode` using
#' [create_hb_cost_test_flags()]. 
#' It will also produce various summary statistics for bedday, cost and
#' episode date variables.
#'
#' @param data new or old data for testing summary flags
#' (data is from \code{\link{get_source_extract_path}})
```